pretext/ExtraTopics/More-BigO.ptx

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-more-bigo">
+  <title>Big O Simplification and Practice</title>
+
+  <p>One more important topic in Big O Analysis is <term>simplification</term>. You will often be asked to do
+    this on assignments. We won't get too deep into the math, but here are a few general rules:</p>
+
+  <p>Constant time operations always simplify to <c>O(1)</c>. You <em>can't</em> have complexities like <c>O(3)</c> or <c>O(1000000)</c>.</p>
```

Comment:
I would say - `O(3)` and `O(1000000)` both simplify to `O(1)` rather than you can't have complexities like...